models/payment/src/payment/payment-draft/presets/sample-data-fashion/payment-aus2.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  LocalizedString,
+  KeyReference,
+  Money,
+} from '@commercetools-test-data/commons';
+import {
+  CustomerDraft,
+  type TCustomerDraft,
+} from '@commercetools-test-data/customer';
+import * as PaymentDraft from '../..';
+import * as PaymentMethodInfo from '../../../../payment-method-info';
+import { PaymentStatusDraft } from '../../../../payment-status';
+import { TransactionDraft } from '../../../../transaction';
+import { TPaymentDraftBuilder } from '../../../types';
+
+const customerAUS = CustomerDraft.presets.sampleDataFashion
+  .sampleUsa()
+  .build<TCustomerDraft>();
+
+const paymentAUS2 = (): TPaymentDraftBuilder =>
+  PaymentDraft.presets
+    .empty()
+    .customer(KeyReference.presets.customer().key(customerAUS.key!))
+    .amountPlanned(Money.random().centAmount(4000).currencyCode('AUD'))
+    .paymentMethodInfo(
+      PaymentMethodInfo.random()
+        .method('Gift Card')
+        .name(LocalizedString.presets.empty()['en']('Gift Card'))
+    )
+    .paymentStatus(PaymentStatusDraft.presets.empty().interfaceText('Paid'))
+    .transactions([
+      TransactionDraft.presets
+        .empty()
+        .amount(Money.random().centAmount(4000).currencyCode('AUD'))
+        .state('Pending')
```

Comment:
Can we make `TransactionState` a constant? Here are the [values](https://docs.commercetools.com/api/projects/payments#transactionstate). The generator can be updated to use one of the constant values, [like so](https://github.com/commercetools/test-data/blob/43f04a398b127e1dd7247fee64f9c0120de48d55/models/order/src/generator.ts#L55).